go/tasks/pluginmachinery/core/resource_manager.go

```diff
@@ -4,21 +4,23 @@ import (
 	"context"
 )
 
-type AllocationStatus string
+//go:generate enumer -type=AllocationStatus -trimprefix=AllocationStatus
+
+type AllocationStatus int
```

Comment:
this would be a breaking change right?

Reply:
No, this is not serialized anywhere... just a return type for the resource manager... I checked